week3/ex3_countdown.py

```diff
@@ -0,0 +1,35 @@
+class Countdown:
```

Comment:
You have two similar files: this one and `countdown.py`.
Which one is correct? Please remove one of them.

Reply:
ex3_countdown is the right one..,countdown.py is removed.

Reply:
> countdown.py is removed.

Is it? I can still see it.

Reply:
now i've deleted it through git,should not be there...,please check.